docs/user/document/tracking/aliases-usage.en.md

```diff
@@ -1,8 +1,10 @@
 # Aliases - Usage
 
-Aliases can be used to indicate the number assigned to this advisory by the coordinator or manufacturer.
+Aliases can be used to indicate the number, identifier or name assigned to this advisory, case or issue by the coordinator, manufacturer, supplier or other party. This can also include internal ticket numbers.
 
```

Comment:
Please add some language to differentiate that from an identifier for a single vulnerability (that should be conveyed with `/vulnerabilities[]/ids`) as this is for document level only.

Reply:
added a sentence on that